### DISCUSSION TITLE
Can you make touchscreen in handheld work as a mouse?

### DISCUSSION THREAD

**Discussion by Crazalu** [Amethyst-szs/smo-lunakit#11] [Requests]
I mean I don't know if its easy or hard, but couldn't you make the touchscreen register with the things in game also the actor browser seems very glitchy with controller

**Comment by Amethyst-szs** [comment]
At the moment I'm not sure! Definitely something I would like to add but that would require a decent bit more work. Definitely on my to do list at some point though!
